README.md

```diff
@@ -1,237 +1,290 @@
+> ### Why do I need to do this?
+
+Any web developer, aspiring or experienced needs to use Git version control, and GitHub is the most popular Git hosting service used by everyone. It is also the heart of the Open Source community. Getting comfortable using GitHub is an essential skill. Making a contribution to a project boosts your confidence and gives you something to show on your GitHub profile.
 
-Any web developer, aspiring or experienced needs to use Git version control, and GitHub is the most popular Git hosting service used by everyone. It is also the heart of the Open Source community. Getting comfortable using GitHub is an essential skill. Making a contribution to a project boosts your confidence and gives you something to show on your GitHub profile.<br>
 If you are a new developer and you are wondering if you need to learn Git and GitHub then here is the answer: [You Should've Learned Git Yesterday](https://codeburst.io/number-one-piece-of-advice-for-new-developers-ddd08abc8bfa 'New Developer? You should’ve learned Git yesterday. by Brandon Morelli, creator of CodeBurst.io').
 
-<h4 align="center">What am I going to contribute?</h4>
+> ### What am I going to contribute?
 
 ![Contributor Card](readme-only/card.PNG 'Contributor Card')
 
 You are going to contribute a card just like this one to this [project's web page](https://syknapse.github.io/Contribute-To-This-Project/ 'https://syknapse.github.io/Contribute-To-This-Project'). It will include your name, your Twitter handle, a short description, and 3 links to useful resources for web developers that you recommend.
 
-You will make a copy of the card template inside the HTML file and customise it with your own information.
+You will make a copy of the card template inside the HTML file and customize it with your own information.
 
-# [If you're comfortable with the terminal go to this tutorial](/terminal-tutorial.md)
+---
 
-<h4 align="center">Translations</h4>
+> ### Translations
 
 This tutorial is also available in [other languages](translations/translation.md)
 
-| [Arabic](translations/README/ARABIC.md)         | [French](translations/README/FRENCH.md)   | [German](translations/README/German.md)   | [Italian](translations/README/ITALIAN.md) | [Korean](translations/README/KOREAN.md) | [Polish](translations/README/POLISH.md) | [Japan](translations/README/JAPANESE.md) |
-| ----------------------------------------------- | ----------------------------------------- | ----------------------------------------- | ----------------------------------------- | --------------------------------------- | --------------------------------------- | --------------------------------------- |
-| [Portuguese](translations/README/PORTUGUESE.md) | [Serbian](translations/README/SERBIAN.md) | [Spanish](translations/README/SPANISH.md) | [Turkish](translations/README/TURKISH.md) | [Bangla](translations/README/BANGLA.md) | [Hindi](translations/README/HINDI.md) | English |
+| 0 | 1 | 2 | 3 | 4 | 5 |
```

Comment:
please remove the horizontal and vertical numbers, they don't really add anything and might be confusing